src/test/kotlin/com/bardsoftware/papeeria/backend/cosmas/PatchCorrectorTest.kt

```diff
@@ -0,0 +1,109 @@
+package com.bardsoftware.papeeria.backend.cosmas
+import org.junit.Assert.*
+import org.junit.Test
+import java.util.*
+import diff_match_patch
+
+class PatchCorrectorTest {
+    @Test
+    fun plusReversePatchTest() {
+        val text1 = "Hello world!"
+        val text2 = "Hello, world!"
+        val dmp = diff_match_patch()
+        val patch = dmp.patch_make(text1, text2)
+        val patchCorrector = PatchCorrector()
+        val listOfPatch = LinkedList<diff_match_patch.Patch>()
+        listOfPatch.addAll(patchCorrector.reversePatch(patch))
```

Comment:
Use [listOf](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/list-of.html)

Reply:
listOf returns MutableList, but diff-match-patch use LinkedList